.codecov.yml

```diff
@@ -31,15 +31,10 @@ comment:
 
 ignore:
   - "LICENSES"
-  - "*_test.go"       
   - ".git"
   - "*.yml"
   - "*.md"
   - "cmd/.*"       
   - "docs/.*"
   - "vendor/.*"
-  - "ddl/failtest/.*"
```

Comment:
Because I am not sure this weather should be ignored.
I think we'd better not ignore them.